embedchain/apps/PersonApp.py

```diff
@@ -22,6 +22,23 @@ def __init__(self, person, config: BaseAppConfig = None):
         self.person_prompt = f"You are {person}. Whatever you say, you will always say in {person} style."  # noqa:E501
         super().__init__(config)
 
+    def add_person_template_to_config(self, default_prompt: str, config: ChatConfig = None):
+
+        self.template = Template(self.person_prompt + " " + default_prompt)
```

Comment:
why use a attribute here? wouldn't a local variable also do the trick. The attribute is a duplicate because it then exists in app and config.

Reply:
updated